.github/workflows/maven.yml

```diff
@@ -11,14 +11,14 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-20.04
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 1.8
+    - name: Set up JDK 11
       uses: actions/setup-java@v1
       with:
-        java-version: 1.8
+        java-version: 11
```

Comment:
I would suggest we keep jdk 1.8 and add jdk 11 in ci, e.g. 

```yaml
jobs:
  build:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        jdk: [7, 8, 11]
    steps:
    - uses: actions/checkout@v2
    - name: Set up JDK
      uses: actions/setup-java@v1
      with:
        java-version: ${{ matrix.jdk }}
```

Reply:
JDK 7 is not supported from version 3.x